panels/user-accounts/org.gnome.controlcenter.user-accounts.gschema.xml.in

```diff
@@ -0,0 +1,12 @@
+<schemalist>
```

Comment:
Is there a reason not to add this to `shell/org.gnome.ControlCenter.gschema.xml`? I don't think any of the other panels have a separate GSettings schema, so it seems like it would be easier to upstream this if we used the main one.

Reply:
I didn't see that gschema, that's the reason :nerd_face: 

I've moved the option there.